shoop/admin/modules/products/views/toolbars.py

```diff
@@ -94,33 +96,66 @@ def _get_variation_and_package_menu_items(self, product):
                     icon="fa fa-eye",
                     url=get_model_url(sib),
                 )
-        elif package_parent:
+
+    def _get_package_menu_items(self, product):
```

Comment:
``_get_package_menu_items`` and ``_get_variation_menu_items`` seems similiar. Can these be combined.